src/spark_rapids_ml/tree.py

```diff
@@ -198,6 +200,12 @@ def _rf_fit(
             rf_params = params[INIT_PARAMETERS_NAME]
             rf_params.pop("n_estimators")
 
+            if rf_params["max_features"] == "auto":
+                if total_trees == 1:
+                    rf_params["max_features"] = 1.0
+                else:
+                    rf_params["max_features"] = "sqrt" if is_classification else 0.33333
```

Comment:
PySpark says 'onethird' in the doc of method trainRegressor(). Is it equivalent to 0.33333 in the implementation?

Reply:
Very close.   Just put 1.0/3.0

Reply:
Done. Thx